src/azul/transformer.py

```diff
@@ -76,12 +78,23 @@ class DocumentCoordinates(NamedTuple):
 FieldTypes = Mapping[str, FieldTypes1]
 
 
+class VersionType(Enum):
+    # No versioning; document is created or overwritten as needed
+    none = 1
+    # Writing a document fails with 409 conflict if one with the same ID already
+    # exists in the index
+    create_only = 2
+    # Use the Elasticsearch "internal" versioning type
```

Comment:
Newline after each item

```suggestion
    
    # Use the Elasticsearch "internal" versioning type
```

Reply:
Added these changes.